spacy_curated_transformers/tests/tokenization/test_registry.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
I don't think we have tests to check the remaining (non-tokenizer) entry points? If that's indeed the case, we should add them too.

Reply:
Yeah, we should move up this file one directory and add the other entry points. Many of them are tested in various places, but it's nice to also have an exhaustive list where we test them all to make sure that nothing slips through the cracks.

Created a ticket for this.